Documentation/Signoffs/Power-Subsystem.md

```diff
@@ -0,0 +1,102 @@
+
+Input Voltage Ripple considerations:
+| # | Peripheral | Ripple Considerations | Source |
+|---|------------|----------------------|--------|
+| 1 | Radar Module | Max ripple 500 mV peak to peak | IPS-937 24GHZ RADAR MODULE datasheet [9] |
+| 2 | RAK811 LPWAN Breakout Module | Max ripple 220 mV peak to peak | RAK811 LPWAN Breakout Module datasheet [10] |
+| 3 | Microphone | Max ripple 500 mV peak to peak | B0B-19389 datasheet [11] |
+| 4 | Speaker | Max ripple 500 mV peak to peak | COM-18343 datasheet [12] |
+
+- Due to the peripherals being powered off of the Jetson's power rails the maximum input voltage ripple is the lowest of the peripherals which is 220 mV peak to peak. 
+
+![Buck converter output voltage ripple](https://github.com/Michaelwwest98/DARPA-Drone-Triage-Sensing-System/assets/123699820/1b264d8e-653e-47e7-ab21-49f261f8b0f4)
+
+- The simulated buck converter circuit above shows an output voltage ripple of around 200 mVp-p that does not dip below 4.75 V which is within standard for the Jetson Nano and all of the peripherals.
+
+5. 
+| # | Peripheral | Power Considerations | Source |
+|---|------------|----------------------|--------|
+| 1 | Radar Module | The minimum supply voltage is 4.75 V. The maximum supply voltage and current is 5.25 V at 65 mA. The typical supply voltage and current is 5.0 V at 55 mA. This gives a typical power consumption of 275 mW and a maximum power consumption of 341.25 mW. Max ripple 500 mV peak to peak | IPS-937 24GHZ RADAR MODULE datasheet [9] |
+| 2 | RAK811 LPWAN Breakout Module | The maximum supply voltage is 3.45 V. The typical supply voltage is 3.3 V. The minimum current consumption is 30 mA when transmitting a signal. Max ripple 220 mV peak to peak | RAK811 LPWAN Breakout Module datasheet [10] |
+| 3 | Microphone | The total power consumption is 4 W. | B0B-19389 datasheet [11] |
+| 4 | Speaker | The maximum supply voltage is 3.6 V. At 3.6 V the device is tested to draw typically 0.265 mA. | COM-18343 datasheet [12] |
+
+The peripherals run off of the Jetson's 5 V pin, 3.3 V pin, and the USB 2.0 ports. The 5 V pins are connected via a power rail. The 3.3 V pin is connected from the 5 V V_in through an MP2152 2A, step-down converter. The overall circuit is essentially the following diagram.
```

Comment:
This is very hand-wavy. I want to see that you have analyzed the circuit to show that this is the case and will work.